src/ray/common/memory_monitor.cc

```diff
@@ -111,41 +111,120 @@ std::tuple<int64_t, int64_t> MemoryMonitor::GetMemoryBytes() {
   return std::tuple(system_used_bytes, system_total_bytes);
 }
 
-int64_t MemoryMonitor::GetCGroupV1MemoryUsedBytes(const char *path) {
-  std::ifstream memstat_ifs(path, std::ios::in | std::ios::binary);
+int64_t MemoryMonitor::GetCGroupV1MemoryUsedBytes(const char *stat_path,
+                                                  const char *usage_path) {
+  // How does this function calculate in-used memory from cgroup memory info file?
```

Comment:
Q: is the implementation aligned with python ones?

Reply:
Sure.